### PR TITLE
See RED-47209, fix crash when using WITHSORTKEYS without SORTBY

### PR DESCRIPTION
**Pull request by MeirShpilraien**

### CODE REVIEW

File: srcutil/code_style.py [hunk 1]
```diff
@@ -106,7 +106,7 @@
     rv = po.wait()
     if rv != 0:
         print('Warning: Could not analyze {}'.format(f))
-    count = len([line for line in output.split('\n') if line])
+    count = len([line for line in output.decode("utf-8").split('\n') if line])
```
Comment (ashtul) [inline srcutil/code_style.py:109]: Is this part of the bug or just an opportunity to fix?

**Review by ashtul — APPROVED** [review]
Nice.

**Comment by ashtul** [comment]
Is this related to 1.6?

File: srcutil/code_style.py (hunk 1 above)
Comment (MeirShpilraien) [reply srcutil/code_style.py:109]: It's just to make it compatible to python 3, I guess you do not install our prehook that check code style..

**Comment by MeirShpilraien** [comment]
> Is this related to 1.6?

If you ask if it happened also on 1.6, then yes.

**Pull request merged**